.github/workflows/ci-cd.yaml

```
@@ -7,52 +7,92 @@ on:
+    steps:
+      - name: Deploy to environment
+        uses: ./.github/actions/deploy-to-environment
+        with:
+          environment: test
+          imageTag: ${{ needs.get-version.outputs.imageTag }}
 
   deploy-staging:
     name: Internal staging
-    uses: ./.github/workflows/deploy-to-environment.yml
+    runs-on: ubuntu-latest
     if: always() && !failure() && !cancelled() 
```

yamllint

[error] trailing spaces

(trailing-spaces)